client/js/lib/helper.js

```diff
@@ -0,0 +1,12 @@
+export const formatDateString = string => {
+  const date = new Date(string);
+
+  let options = {
+    month: 'short',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+  };
+
+  return date.toLocaleDateString('en-us', options).replace(',', ' at');
+};
```

Comment:
use moment.js.

Reply:
We considered it but this is the only place we're formatting a date and we think there's no point installing an entire package (and a fairly large one as well) for one function, especially given we have a solution already.